asm/non_matchings/code_80005FD0/update_player_path_completion.s

```diff
@@ -233,20 +233,20 @@ glabel func_8000929C
 /* 00A1DC 800095DC A5AC0000 */  sh    $t4, ($t5)
 /* 00A1E0 800095E0 A7A80034 */  sh    $t0, 0x34($sp)
 /* 00A1E4 800095E4 AFA70030 */  sw    $a3, 0x30($sp)
-/* 00A1E8 800095E8 0C0023CE */  jal   func_80008F38
+/* 00A1E8 800095E8 0C0023CE */  jal   update_player_completion
 /* 00A1EC 800095EC AFA40050 */   sw    $a0, 0x50($sp)
 /* 00A1F0 800095F0 0C0046B9 */  jal   reset_kart_ai_behaviour
 /* 00A1F4 800095F4 8FA40050 */   lw    $a0, 0x50($sp)
 /* 00A1F8 800095F8 8FA40050 */  lw    $a0, 0x50($sp)
 /* 00A1FC 800095FC 3C0F8016 */  lui   $t7, %hi(D_8016348C) # $t7, 0x8016
 /* 00A200 80009600 85EF348C */  lh    $t7, %lo(D_8016348C)($t7)
-/* 00A204 80009604 3C018016 */  lui   $at, %hi(D_801642D8 + 0x6) # 0x8016
+/* 00A204 80009604 3C018016 */  lui   $at, %hi(gCpuItemStrategy + 0x6) # 0x8016
```

Comment:
gCPU ?

Reply:
by memory once I name a variable with CPU in uppercase but you say no so I rename with cpu in pascal case